.gitignore

```diff
@@ -1,5 +1,7 @@
 node_modules/
 dist/
-lib/
+## Use trailing slash, to track /server/lib
+## @see https://stackoverflow.com/a/26327728
+/lib/
```

Comment:
I've generally seen `lib/` used for compiled code (i.e. for use as a library), and `src` for actual source (e.g. `react` [source](https://github.com/facebook/react/tree/master/packages/react) vs. [package contents on npm](https://unpkg.com/react@15.6.2/)) and we follow that pattern in our modules.

Would it make sense to store this in `server/src` for consistency?